private-templates-service/client/src/App.tsx

```diff
@@ -1,40 +1,59 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import { Container } from "reactstrap";
-
 import { UserAgentApplication, ClientAuthError } from "msal";
 
+// Redux
+import { connect } from "react-redux";
+import { login, logout } from "./store/auth/actions";
+import { UserType } from "./store/auth/types";
+import { RootState } from "./store/rootReducer";
+
+// Components
 import NavBar from "./components/NavBar/NavBar";
 import ErrorMessage from "./components/ErrorMessage/ErrorMessage";
 import Welcome from "./components/Welcome/Welcome";
 import config from "./Config";
 import { getUserDetails, getOrgDetails } from "./Services/GraphService";
 import { ErrorMessageProps } from "./components/ErrorMessage/ErrorMessage";
 
+// CSS
 import "bootstrap/dist/css/bootstrap.css";
```

Comment:
Are we still using bootstrap?

Reply:
no but that is outside the scope of this PR